### PR TITLE
Add Sponsorship field to Rich Links

### PR DESCRIPTION
**Pull request by katebee**
Related:

https://github.com/guardian/content-api/pull/2030

### CODE REVIEW

File: models/src/main/thrift/content/v1.thrift [hunk 1]
```diff
@@ -559,6 +559,7 @@ struct RichLinkElementFields {
     3: optional string linkText
     4: optional string linkPrefix
     5: optional string role
+    6: optional Sponsorship sponsorship
```
Comment (tomrf1) [inline models/src/main/thrift/content/v1.thrift]: Is this intended to match the [model in flexible-model](https://github.com/guardian/flexible-model/blob/master/src/main/thrift/content.thrift#L403)?
Unfortunately this library doesn't depend on flexible-model, so you'll need to copy the thrift definition in here

Comment (katebee) [reply models/src/main/thrift/content/v1.thrift]: ??? 🤔 

I believe `RichLinkElementFields` is now using this version of [Sponsorship](https://github.com/guardian/content-api-models/blob/fa31a09b0a84a8cfc754ac43e39bffae7589b29a/models/src/main/thrift/content/v1.thrift#L1076), as a transformation occurs in content-api

@LATaylor-guardian just a thought - will it be a problem that `Sponsorship` is defined below the `RichLinkElementFields`?

Comment (tomrf1) [reply models/src/main/thrift/content/v1.thrift]: sorry I completely missed the point here :)

Comment (tomrf1) [reply models/src/main/thrift/content/v1.thrift]: The build is currently failing, I think you're right that it's because `Sponsorship` is defined after

Comment (LATaylor-guardian) [reply models/src/main/thrift/content/v1.thrift]: @katebee Yes, good spot. This is why it's not compiling.

**Comment by tomrf1** [comment]
👍 I'll release this shortly

**Pull request merged**